x/ccv/consumer/module.go

```diff
@@ -92,16 +93,22 @@ func (AppModuleBasic) GetQueryCmd() *cobra.Command {
 	return cli.NewQueryCmd()
 }
 
+type StakingKeeper interface {
```

Comment:
Do we need a new staking keeper interface? Or could you make use of existing ones from `x/ccv/types/expected_keepers.go` or `testutil/e2e/interfaces.go`,

Reply:
Will need to check

Reply:
Existing one is to be used on provider side, and this one is for consumer side and I think it would be better to not merge two into one. And the required function for StakingKeeper is different from ones from `x/ccv/types/expected_keepers.go` or `testutil/e2e/interfaces.go`

Reply:
Good point, I agree that it's useful to have two separate interfaces for what the consumer and provider expect as the staking keeper. This interface should exist in the same file as the provider's interface tho, ie. `x/ccv/types/expected_keepers.go`

Reply:
Okay

Reply:
It's put as `x/ccv/types.DemocracyStakingKeeper`